src/ImageSequencer.js

```diff
@@ -46,6 +46,13 @@ ImageSequencer = function ImageSequencer(options) {
   events = require('./ui/UserInterface')(),
   fs = require('fs');
 
+
+  if(options.inBrowser){
```

Comment:
Here, i think a code cleaning sweep would be helpful to ensure really consistent code formatting. I'd like to see this be:

`if (options.inBrowser) {` <= note the spaces. Also, i see a lot of examples of comma use like `[param,param]` -- can we standardize on `[param, param]` and `function(param, param) {`? This should be configurable in your IDE. Sorry to nitpick about things like this but having a clear standard is really good for long term code tidiness -- we want your excellent new work to last a long time!!! :smile:

Reply:
@jywarren Oh I get it. No problem, I'll configure this right now!! Thanks.

Reply:
done